crates/proto/src/op/message.rs

```diff
@@ -72,7 +72,7 @@ pub struct Message {
     answers: Vec<Record>,
     name_servers: Vec<Record>,
     additionals: Vec<Record>,
-    signature: Vec<Record>,
+    signature: MessageSignature,
```

Comment:
Do we have a reference for multiple signatures never being allowed/needed?

Reply:
Ahh, nice, that makes sense then. Can you add some of these references in the code somewhere?

Reply:
Sure, maybe in `read_records()` where it's enforced? :thinking:

Reply:
Added some mentions to `read_records()` and a pointer from the `MessageSignature` struct rustdoc.